src/bsoncxx/CMakeLists.txt

```diff
@@ -158,10 +158,7 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
     DESTINATION ${BSONCXX_HEADER_INSTALL_DIR}
     COMPONENT dev
     FILES_MATCHING
-      PATTERN "*.h"
```

Comment:
I checked and the only .h header was a private one.  From Drews comment yesterday, it sounds like we want to limit ourselves to .hpp.